encryption-service-jasypt/src/main/java/com/mx/path/service/facility/encryption/jasypt/JasyptEncryptionService.java

```diff
@@ -58,10 +60,12 @@ public class JasyptEncryptionService implements EncryptionService {
   // Fields
 
   @Getter
-  private ObjectMap configurations;
+  private final ObjectMap configurations;
```

Comment:
Why are we making configurations final?
Also what's the advance to set accessLevel as package?

Reply:
Making final because the reference to the configs can never be modified after construction. Making package level on a couple of attributes for testing purposes.